### PR TITLE
Initial setup of resume

### PR DESCRIPTION
**Pull request by aneeshs005**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1 +1,342 @@
-"Hello World" 
+<!DOCTYPE html>
+<html lang="en"> 
+<head>
+    <title>Aneesh S - Bootstrap 4 Resume/CV Developer</title>
```
Comment (sreeragps) [inline index.html]: change title to Aneesh S Resume it will be better

File: index.html [hunk 2]
```diff
@@ -1 +1,342 @@
...
+    
+    <!-- Meta -->
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="Responsive Resume Template">
```
Comment (sreeragps) [inline index.html]: change this

File: index.html [hunk 3]
```diff
@@ -1 +1,342 @@
...
+    <meta name="author" content="Xiaoying Riley at 3rd Wave Media">
```
Comment (sreeragps) [inline index.html]: change it too

File: index.html [hunk 4]
```diff
@@ -1 +1,342 @@
...
+						    <ul class="list-unstyled">
+							    <li class="mb-2"><a href="#"><i class="far fa-envelope fa-fw mr-2" data-fa-transform="grow-3"></i>AneeshSXXX@gmail.com</a></li>
+							    <li><a href="#"><i class="fas fa-mobile-alt fa-fw mr-2" data-fa-transform="grow-6"></i>+91 949552XXXX</a></li>
+						    </ul>
+					    </div><!--//primary-info-->
+					    <div class="secondary-info ml-md-auto mt-2">
+						    <ul class="resume-social list-unstyled">
+				                <li class="mb-3"><a href="#"><span class="fa-container text-center mr-2"><i class="fab fa-linkedin-in fa-fw"></i></span>linkedin.com/in/aneeshs005</a></li>
+				                <li class="mb-3"><a href="#"><span class="fa-container text-center mr-2"><i class="fab fa-github-alt fa-fw"></i></span>https://github.com/aneeshs005</a></li>
+				                <!--<li class="mb-3"><a href="#"><span class="fa-container text-center mr-2"><i class="fab fa-codepen fa-fw"></i></span>codepen.io/username/</a></li>-->
```
Comment (sreeragps) [inline index.html]: remove unneeded code

**Review by sreeragps — CHANGES_REQUESTED** [review]
please fix this

File: index.html [hunk 5]
```diff
@@ -1 +1,342 @@
...
+									    <div class="resume-timeline-item-desc">
+											<ul>
+											    <li>Providing support for technicians dispatched on jobs related to Internet, Cvoip, IPTV and DTV.</li>
+											    <li>Assisted with changing bandwidth and profile for internet and IPTV services.</li>
+											    <li>Fixing sync issue with Modem and surfing, troubleshooting IPTV issues.</li>
+											    <li>Provisioning DTV and IPTV services.</li>
+											</ul>
+											<!--
+										    <p>Role description goes here ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo ligula eget dolor. Aenean massa. Cum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Donec quam felis, ultricies nec, pellentesque eu, pretium quis, sem. Donec pede justo, fringilla vel.</p>
+										    <h4 class="resume-timeline-item-desc-heading font-weight-bold">Achievements</h4>
```
Comment (sreeragps) [inline index.html]: delete this commented codes

File: index.html [hunk 6]
```diff
@@ -1 +1,342 @@
...
+									    <div class="resume-timeline-item-header mb-2">
+										    <div class="d-flex flex-column flex-md-row">
+										        <h3 class="resume-position-title font-weight-bold mb-1">Senior Specialist Technical Support</h3>
+										        <div class="resume-company-name ml-auto">Sutherland Global Services, Kochi</div>
+										    </div><!--//row-->
+										    <div class="resume-position-time">12/2012 - 02/2016</div>
+									    </div><!--//resume-timeline-item-header-->
+									    <div class="resume-timeline-item-desc">
+											<!--
+										    <p>Role description goes here ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo ligula eget dolor. Aenean massa. Cum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Donec quam felis, ultricies nec.</p>
```
Comment (sreeragps) [inline index.html]: delete unwanted codes

File: index.html [hunk 7]
```diff
@@ -1 +1,342 @@
...
+
+								    </article><!--//resume-timeline-item-->
+								</div><!--//resume-timeline-->
+							    
+						    </div>
+					    </section><!--//experience-section-->
+				    </div>
+				    <div class="col-lg-3">
+					    <section class="resume-section skills-section mb-5">
+						    <h2 class="resume-section-title text-uppercase font-weight-bold pb-3 mb-3">Skills &amp; Tools</h2>
```
Comment (sreeragps) [inline index.html:149]: put proper header

File: index.html [hunk 8]
```diff
@@ -1 +1,342 @@
...
+						    <div class="resume-section-content">
+							    <ul class="list-unstyled">
+								    <li class="mb-2">
+								        <div class="resume-degree font-weight-bold">B.Tech Electronics and Communication</div>
+								        <div class="resume-degree-org">Cochin University of Science and Technology</div>
+								        <div class="resume-degree-time">2006-Pursuing</div>
+								    </li>
+								    <li>
+								        <div class="resume-degree font-weight-bold">ASP.NET Full Stack Development Course</div>
+								        <div class="resume-degree-org">STEPS, Kochi</div>
```
Comment (sreeragps) [inline index.html:245]: change STEPS

File: index.html [hunk 9]
```diff
@@ -1 +1,342 @@
...
+			    </div><!--//row-->
+		    </div><!--//resume-body-->
+		    
+		    
+	    </div>
+    </article>  
+
+    
+    <footer class="footer text-center pt-2 pb-5">
+	    <!--/* This template is free as long as you keep the footer attribution link. If you'd like to use the template without the attribution link, you can buy the commercial license via our website: themes.3rdwavemedia.com Thank you for your support. :) */-->
```
Comment (sreeragps) [inline index.html]: Delete the text

File: index.html [hunk 10]
```diff
@@ -1 +1,342 @@
...
+        <small class="copyright">Designed with <i class="fas fa-heart"></i> by <a href="http://themes.3rdwavemedia.com" target="_blank">Xiaoying Riley</a> for developers</small>
```
Comment (sreeragps) [inline index.html]: change Xiaoying Riley to Brainchild Services

**Review by sreeragps — CHANGES_REQUESTED** [review]
check on & take necessary changes

File: index.html (hunk 7 above)
Comment (aneeshs005) [reply index.html:149]: What is the issue with this header?

File: index.html (hunk 8 above)
Comment (aneeshs005) [reply index.html:245]: STEPS is the name of the institution, why should it be removed?

File: index.html (hunk 9 above)
Comment (aneeshs005) [reply index.html]: Since its a free license, i think we need to keep the copy right code.

File: index.html (hunk 10 above)
Comment (aneeshs005) [reply index.html]: The license issue applies here as well

**Pull request merged**